aws-android-sdk-iot-test/src/androidTest/java/com/amazonaws/mobileconnectors/iot/MqttManagerIntegrationTest.java

```diff
@@ -747,45 +719,46 @@ public void onFailure(Throwable exception) {
 
     @Test
     public void testWebSocketConnectionWithIamAuth() throws Exception {
-        testWebsocketConnection(AWSIotMqttManager.AuthenticationMode.IAM, endpoint);
+        testWebsocketConnection(AuthenticationMode.IAM, iotClient.getEndpointAddress());
     }
 
     @Test
     public void testWebSocketConnectionWithCustomAuth() throws Exception {
-        testWebsocketConnection(AWSIotMqttManager.AuthenticationMode.CUSTOM_AUTH, endpoint);
+        testWebsocketConnection(AuthenticationMode.CUSTOM_AUTH, iotClient.getEndpointAddress());
     }
 
     @Test
     public void testWebSocketWithConnectionUsernamePassword() throws Exception {
-        testWebsocketConnection(AWSIotMqttManager.AuthenticationMode.USERNAME_PASSWORD, betaEndpoint);
+        String betaEndpoint = iotClient.getEndpointAddress("iot:Data-Beta");
+        testWebsocketConnection(AuthenticationMode.USERNAME_PASSWORD, betaEndpoint);
     }
 
     @Test
     public void testWebSocketWithConnectionIamAuthAndCustomDomain() throws Exception {
-        testWebsocketConnection(AWSIotMqttManager.AuthenticationMode.IAM, CUSTOM_DOMAIN);
+        testWebsocketConnection(AuthenticationMode.IAM, CUSTOM_DOMAIN);
     }
 
     @Test
     public void testWebSocketConnectionWithCustomAuthAndCustomDomain() throws Exception {
-        testWebsocketConnection(AWSIotMqttManager.AuthenticationMode.CUSTOM_AUTH, CUSTOM_DOMAIN);
+        testWebsocketConnection(AuthenticationMode.CUSTOM_AUTH, CUSTOM_DOMAIN);
     }
 
     @Test
     public void testWebSocketConnectionWithUsernamePasswordAndCustomDomain() throws Exception {
-        testWebsocketConnection(AWSIotMqttManager.AuthenticationMode.USERNAME_PASSWORD, CUSTOM_DOMAIN);
+        testWebsocketConnection(AuthenticationMode.USERNAME_PASSWORD, CUSTOM_DOMAIN);
     }
 
     private void testWebsocketConnection(
-            final AWSIotMqttManager.AuthenticationMode authMode,
+            final AuthenticationMode authMode,
             final String endpoint)
             throws InterruptedException, JSONException {
         final List<AWSIotMqttClientStatusCallback.AWSIotMqttClientStatus> statuses = new ArrayList<>();
         final List<String> messages = new ArrayList<>();
 
         AWSIotMqttManager mqttManager = AWSIotMqttManager.from(
             Region.getRegion(Regions.US_EAST_1),
```

Comment:
I can't fight this squeamishness about having this constant everywhere. Another time.